src/Sentry.Unity.Android/SentryNativeAndroid.cs

```diff
@@ -11,7 +12,7 @@ public static class SentryNativeAndroid
         /// Configures the native Android support.
         /// </summary>
         /// <param name="options">The Sentry Unity options to use.</param>
-        public static void Configure(SentryUnityOptions options)
+        public static void Configure(SentryUnityOptions options, bool isIL2CPP = false)
```

Comment:
If you take suggestion above, then we'd always require the value:

```suggestion
        public static void Configure(SentryUnityOptions options, bool il2cpp)
```